src/components/ProductListItem.vue

```diff
@@ -6,6 +6,7 @@
     <ion-label>
       <h5>{{ item.brandName }}</h5>
       <h2>{{ item.itemName }}</h2>
+      <h2>{{ getProduct(item.itemId).internalName }}</h2>
```

Comment:
Use p tag here for internal name and add ion-text-wrap utility class. Verify class name. I may be wrong.